expression/testdata/partition_pruner_out.json

```diff
@@ -89,18 +89,6 @@
         "Result": [
           "TableDual_8 0.00 root rows:0"
         ]
-      },
-      {
-        "SQL": "explain select * from t1 where t1.a > 7 and t1.a < 3",
```

Comment:
ping @tiancaiamao

Reply:
Because I remove constraint propagate solver, now `a > 7 and a < 3` can't be solved.
This plan now change to union all partitions.

I guess it's acceptable: our user should be clever enough and not writing this kind of silly queries.